<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="1" ignored="0" total="8" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Stock Suite" duration-ms="460" started-at="2018-09-13T01:26:30Z" finished-at="2018-09-13T01:26:30Z">
    <groups>
    </groups>
    <test name="Add Delete Stock Test" duration-ms="208" started-at="2018-09-13T01:26:30Z" finished-at="2018-09-13T01:26:30Z">
      <class name="com.selenium.testcases.stocksuite.AddDelStockTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f]" name="init" is-config="true" duration-ms="132" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f]" name="initTest" is-config="true" duration-ms="0" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="SKIP" signature="addStockTest(java.util.Hashtable)[pri:1, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f]" name="addStockTest" duration-ms="2" started-at="2018-09-12T21:26:30Z" data-provider="getData" finished-at="2018-09-12T21:26:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedStockName=Tata Steel, Date=8/19/2018, PurchasePrice=200, PortfolioName=Manish, Runmode=Y, Quantity=100, StockName=Tata Steel Ltd, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to NO
	at com.selenium.testcases.stocksuite.AddDelStockTest.addStockTest(AddDelStockTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addStockTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f]" name="quit" is-config="true" duration-ms="67" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="SKIP" signature="deleteStockTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f]" name="deleteStockTest" duration-ms="0" started-at="2018-09-12T21:26:30Z" data-provider="getData" depends-on-methods="com.selenium.testcases.stocksuite.AddDelStockTest.addStockTest" finished-at="2018-09-12T21:26:30Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AddDelStockTest.deleteStockTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AddDelStockTest.deleteStockTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.stocksuite.AddDelStockTest@2f686d1f] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteStockTest -->
      </class> <!-- com.selenium.testcases.stocksuite.AddDelStockTest -->
    </test> <!-- Add Delete Stock Test -->
    <test name="Portfolio Test" duration-ms="246" started-at="2018-09-13T01:26:30Z" finished-at="2018-09-13T01:26:30Z">
      <class name="com.selenium.testcases.stocksuite.BuySellStockTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="init" is-config="true" duration-ms="105" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="buyOrSellStockTest(java.util.Hashtable)[pri:1, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="buyOrSellStockTest" duration-ms="3" started-at="2018-09-12T21:26:30Z" data-provider="getData" finished-at="2018-09-12T21:26:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Date=8/19/2018, Action=Buy, PurchasePrice=200, PortfolioName=Manish, Runmode=Y, Quantity=100, StockName=Tata Motors Ltd., Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to NO
	at com.selenium.testcases.stocksuite.BuySellStockTest.buyOrSellStockTest(BuySellStockTest.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyOrSellStockTest -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="initTest" is-config="true" duration-ms="0" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="quit" is-config="true" duration-ms="63" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="SKIP" signature="checkTransactionHistoryTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="checkTransactionHistoryTest" duration-ms="2" started-at="2018-09-12T21:26:30Z" data-provider="getData" finished-at="2018-09-12T21:26:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{PortfolioName=Manish, Runmode=Y, StockName=Tata Motors Ltd., Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to NO
	at com.selenium.testcases.stocksuite.BuySellStockTest.checkTransactionHistoryTest(BuySellStockTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTransactionHistoryTest -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="initTest" is-config="true" duration-ms="0" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.selenium.testcases.stocksuite.BuySellStockTest@5f2108b5]" name="quit" is-config="true" duration-ms="67" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.selenium.testcases.stocksuite.BuySellStockTest -->
    </test> <!-- Portfolio Test -->
  </suite> <!-- Stock Suite -->
  <suite name="My suite" duration-ms="0" started-at="2018-09-13T01:26:30Z" finished-at="2018-09-13T01:26:30Z">
    <groups>
    </groups>
  </suite> <!-- My suite -->
  <suite name="Portfolio Suite" duration-ms="79876" started-at="2018-09-13T01:25:10Z" finished-at="2018-09-13T01:26:30Z">
    <groups>
    </groups>
    <test name="Login Test" duration-ms="79593" started-at="2018-09-13T01:25:10Z" finished-at="2018-09-13T01:26:29Z">
      <class name="com.selenium.testcases.portfoliosuite.LoginTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="init" is-config="true" duration-ms="1475" started-at="2018-09-12T21:25:10Z" finished-at="2018-09-12T21:25:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="initTest" is-config="true" duration-ms="358" started-at="2018-09-12T21:25:11Z" finished-at="2018-09-12T21:25:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="PASS" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="loginTest" duration-ms="74298" started-at="2018-09-12T21:25:12Z" data-provider="getData" finished-at="2018-09-12T21:26:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=manish_chaudhari18, ExpectedResult=Loginsuccess, Password=ManishSarita, Runmode=Y, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="quit" is-config="true" duration-ms="1273" started-at="2018-09-12T21:26:26Z" finished-at="2018-09-12T21:26:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="FAIL" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="loginTest" duration-ms="1971" started-at="2018-09-12T21:26:27Z" data-provider="getData" finished-at="2018-09-12T21:26:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=manish_chaudhari18, ExpectedResult=Loginsuccess, Password=ManishSarita, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.selenium.driver.DriverScript.executeKeywords(DriverScript.java:46)
	at com.selenium.testcases.portfoliosuite.LoginTest.loginTest(LoginTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status 0
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'MANISH-PC', ip: '192.168.1.237', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_181'
Driver info: driver.version: FirefoxDriver
remote stacktrace: 
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$new$0(W3CHandshakeResponse.java:57)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$getResponseFunction$2(W3CHandshakeResponse.java:104)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:123)
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
	at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:126)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:98)
	at com.selenium.keywords.GenericKeywords.openBrowser(GenericKeywords.java:128)
	... 32 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="initTest" is-config="true" duration-ms="0" started-at="2018-09-12T21:26:27Z" finished-at="2018-09-12T21:26:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.selenium.testcases.portfoliosuite.LoginTest@4883b407]" name="quit" is-config="true" duration-ms="96" started-at="2018-09-12T21:26:29Z" finished-at="2018-09-12T21:26:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.selenium.testcases.portfoliosuite.LoginTest -->
    </test> <!-- Login Test -->
    <test name="Portfolio Test" duration-ms="243" started-at="2018-09-13T01:26:29Z" finished-at="2018-09-13T01:26:30Z">
      <class name="com.selenium.testcases.portfoliosuite.PortFolioTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c]" name="init" is-config="true" duration-ms="142" started-at="2018-09-12T21:26:29Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="createPortFolioTest(java.util.Hashtable)[pri:1, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c]" name="createPortFolioTest" duration-ms="3" started-at="2018-09-12T21:26:30Z" data-provider="getData" finished-at="2018-09-12T21:26:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{PortfolioName=TestingPortfolio1, Runmode=Y, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to NO
	at com.selenium.testcases.portfoliosuite.PortFolioTest.createPortFolioTest(PortFolioTest.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPortFolioTest -->
        <test-method status="PASS" signature="initTest()[pri:0, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c]" name="initTest" is-config="true" duration-ms="0" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c]" name="quit" is-config="true" duration-ms="74" started-at="2018-09-12T21:26:30Z" finished-at="2018-09-12T21:26:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="SKIP" signature="deletePortFolioTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c]" name="deletePortFolioTest" duration-ms="0" started-at="2018-09-12T21:26:30Z" data-provider="getData" depends-on-methods="com.selenium.testcases.portfoliosuite.PortFolioTest.createPortFolioTest" finished-at="2018-09-12T21:26:30Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method PortFolioTest.deletePortFolioTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method PortFolioTest.deletePortFolioTest(java.util.Hashtable)[pri:2, instance:com.selenium.testcases.portfoliosuite.PortFolioTest@553f17c] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1289)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletePortFolioTest -->
      </class> <!-- com.selenium.testcases.portfoliosuite.PortFolioTest -->
    </test> <!-- Portfolio Test -->
  </suite> <!-- Portfolio Suite -->
</testng-results>
